src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -1065,10 +1065,15 @@ bool extend_reverse_type(const std::shared_ptr<ov::Node>& node, const precisions
 
 template <typename src_type, typename dst_type>
 inline dst_type convert_value(src_type val) {
-    if (val > std::numeric_limits<dst_type>::max()) {
-        return std::numeric_limits<dst_type>::max();
-    } else if (val < std::numeric_limits<dst_type>::lowest()) {
-        return std::numeric_limits<dst_type>::lowest();
+    if (std::numeric_limits<src_type>::max() > std::numeric_limits<dst_type>::max()) {
```

Comment:
probably it should constexpr to resolve the coverity issue
but our bfloat16 impl doesn't have constexpr > and < operators overloaded

Reply:
decided to move it to the separate PR